src/main/java/com/semmle/jira/addon/Request.java

```diff
@@ -182,34 +183,52 @@ boolean isValid() {
 
     @JsonIgnoreProperties(ignoreUnknown = true)
     public static class Query {
-      @JsonProperty public final String name;
+      @Deprecated @JsonProperty private final String name;
+      @JsonProperty public final String language;
```

Comment:
The oposite of the name here. The language is not mandatory in the `isValid()` function, so I think it should be inside the properties object.

Reply:
The properties object corresponds to the query metadata specified in the header comment of a `.ql` file. For example `@name` and `@id` . Language is not a metadata property in that sense.